src/domain/events/utils.ts

```diff
@@ -267,6 +269,57 @@ export const getEventDateText = (
   return '-';
 };
 
+export const getEventTimeStr = ({
```

Comment:
This function could use a bit of refactoring into smaller and more focused functions per each case. Also `timeFormat` could be initialized with a separate function,

Reply:
Added a separate function to get timeFormat. Also moved logic to format time string in the case that both start time and end time are defined to a separate function